### PR TITLE
SCT filtering + find/replace

### PR DESCRIPTION
**Pull request by taroface**
DOC-8325

- Document SCT filtering.
- Add note on Replace All functionality.
- Slightly simplify layout of SCT doc.

### CODE REVIEW

File: src/current/cockroachcloud/migrations-page.md [hunk 1]
```diff
@@ -253,22 +245,43 @@ To edit a statement, click the **Edit** button or the statement itself and enter
...
+- [Suggestions](#suggestions)
+- [Compatibility Notes](#compatibility-notes) (non-PostgreSQL schemas)
+
+{{site.data.alerts.callout_info}}
+If a filter is not available, `No Options` is displayed.
+{{site.data.alerts.end}}
+
+Click **Apply** to filter the **Statements** list using the selected filters.
+
+To remove a filter, click the `x` next to the active filter. To remove all active filters, click **Clear filters** above the **Statements** list.
```
Comment (ryanluu12345) [inline src/current/cockroachcloud/migrations-page.md:264]: "To remove an active filter, click the `x` next to the active filter name."

**Comment by ryanluu12345** [comment]
It generally looks good to me. However I don't see the removal of the docs for "Collapse statements". We no longer support collapsing statements now that we have filtering.

Also, I'm not sure I follow, but why did we "Add note on Replace All functionality". I don't think anything changed in that regard.

**Comment by taroface** [comment]
> It generally looks good to me. However I don't see the removal of the docs for "Collapse statements". We no longer support collapsing statements now that we have filtering.
> 
> Also, I'm not sure I follow, but why did we "Add note on Replace All functionality". I don't think anything changed in that regard.

TFTR! There was a mention of "Collapse statements" that was removed in this doc. And as discussed separately, we hadn't documented Replace All, so that was added here.

**Review by Amruta-Ranade — APPROVED** [review]
LGTM!

**Comment by ryanluu12345** [comment]
LGTM now! Thanks for addressing :)

**Pull request merged**